loopout.go

```diff
@@ -647,8 +650,8 @@ func (s *loopOutSwap) payInvoice(ctx context.Context, invoice string,
 		var result paymentResult
 
 		status, err := s.payInvoiceAsync(
```

Comment:
If we follow `s.payInvoiceAsync` and the `paymentTimeout` it ends up in `awaitSendPayment`. The context here is passed to `s.lnd.Router.SendPayment(ctx, *req)`.
Question: Does it make sense to cancel this context once the `paymentTimeout` is reached, given that we think about making this context cancelable to properly shutdown `lnd`'s payment loop? The related PR is here: https://github.com/lightningnetwork/lnd/pull/8734

Reply:
Yeah I think this could be a good idea, but given that not all users switch to the latest LND release, we need to still rely on the request's timeout. For now I'll keep it as is and we can add it in a separate PR once we bump the minimum LND version.